content/en/monitors/guide/monitoring-sparse-metrics.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Monitoring Sparse Metrics
+kind: guide
+further_reading:
+- link: "/api/latest/monitors/#edit-a-monitor"
+  tag: "Documentation"
+  text: "Learn more about updating these settings via API"
+- link: "/dashboards/functions/interpolation/#default-zero"
+  tag: "Documentation"
+  text: "Learn more about interpolation"
+---
+
+## Overview
+
+When monitoring a metric that reports data infrequently, there are some tools and behaviors that should be considered to ensure a monitor's settings are appropriate for evaluating these queries as intended. 
+
+## Source of the monitor
+
+First, consider your source.
+
+### Metric-based monitor
+
+Is this a metric, change, anomaly or other metric-based monitor ? The following settings should be adjusted:
+
+* Make sure that **Do not require a full window of data for evaluation** is set
+* Is the data often delayed? Consider adjusting the monitor evaluation delay as well
+* Adjust the evaluation (average, max, min, sum) based on the expected frequency. The default evaluation is **average**, which may not be suited for sparse metrics.
+* If you are using **average**, think about adding an [interpolation function][1] like `default_zero()` to ensure the gaps in the metric are evaluated as zero.
+* If you are using arithmetic in your query, take a look at [Monitor Arithmetic and Sparse Metrics][2] for some further guidance.
+
+
+### Event-based Monitor 
+
+Is this a log, event or other event-based monitor? Look at the following:
+
+* Make sure that the "Missing data" setting corresponds to your expected monitor behavior: **Evaluate as zero**, **Show NO DATA**, **Show NO DATA and notify**, or **Show OK**
+* Adjust the evaluation period, if data points are expected to be available every 30 minutes, then the evaluation period should account for that.
+
+
+## Scheduled-based Monitoring
+
+Are you monitoring for an event that needs to happen at certain times of the day, week, month? A CRON task (backup job, export, etc.)? Then you may want to look at [Custom Schedules][3] instead, which allow you to set RRULES to define when the monitor should evaluate and notify.
+
+## Tip: How to determine whether you have a sparse metric
```

Comment:
```suggestion
## How to determine whether you have a sparse metric
```

Reply:
Optional, I think this reads better as the first section of this guide rather than the last.